app/controllers/submit.js

```diff
@@ -1133,68 +1208,35 @@ export default Controller.extend(Analytics, BasicsValidations, NodeActionsMixin,
         }
     },
 
-    _setNodePreprints(preprints) {
-        this.set('existingPreprints', preprints);
-        if (preprints.toArray().length > 0) { // If node already has a preprint
-            const preprint = preprints.toArray()[0]; // TODO change after branded finished
-            if (!(preprint.get('isPublished'))) { // Preprint exists in abandoned state.
-                this.set('abandonedPreprint', preprint);
-            }
-        }
-    },
-
-    _sendToPreprintStartOrAbandon() {
-        this.send(this.get('abandonedPreprint') ? 'resumeAbandonedPreprint' : 'startPreprint');
-    },
-
-    _setNodeTitle() {
-        const node = this.get('node');
-        const model = this.get('model');
-
-        const currentNodeTitle = node.get('title');
-        const title = this.get('title');
-
-        if (currentNodeTitle === title) {
-            return;
-        }
-        model.set('title', title);
-        node.set('title', title);
-        return node.save();
-    },
-
-    _failSetNodeTitle() {
-        const node = this.get('node');
-        const currentNodeTitle = node.get('title');
-
-        node.set('title', currentNodeTitle);
-        this.get('toast').error(this.get('i18n').t('submit.could_not_update_title'));
-    },
-
-    _addChild(child) {
-        const node = this.get('node');
-
-        this.set('parentNode', node);
-        this.send('getParentContributors', node);
-        this.set('node', child);
-        this.set('basicsAbstract', this.get('node.description') || null);
-        child.get('files')
-            .then(this._getFiles.bind(this))
+    _getFileProviders() {
+        const preprint = this.get('model');
+        preprint.get('files')
+            .then(this._copyNodeFileToPreprint.bind(this))
             .catch(this._failGetFiles.bind(this));
     },
 
-    _getFiles(fileProviders) {
-        const child = this.get('node');
+    _copyNodeFileToPreprint(fileProviders) {
+        const preprint = this.get('model');
         const osfstorage = fileProviders.findBy('name', 'osfstorage');
 
-        this.get('fileManager').copy(this.get('selectedFile'), osfstorage, { data: { resource: child.id } })
-            .then(this._copyFile.bind(this))
+        this.get('fileManager').copy(this.get('selectedFile'), osfstorage, { data: { resource: preprint.id, provider: 'osfstorage', conflict: 'replace' } })
+            .then(this._setPrimaryFileAndNodeAttributes.bind(this))
             .catch(this._failCopyFile.bind(this));
     },
 
-    _copyFile(copiedFile) {
+    _setPrimaryFileAndNodeAttributes(copiedFile) {
```

Comment:
if you pick your preprint file from an existing node, that node's tags, description, and contributors are copied over for you to edit.  They are saved immediately, rather than staged as pending changes per the general save per section that is in place.